docs/admin/config.rst

```diff
@@ -788,7 +817,7 @@ List of enabled machine translation services to use.
 
 .. note::
 
-    Many of services need additional configuration like API keys, please check
+    Many of the services need additional configuration like API keys, please check
     their documentation for more details.
```

Comment:
There should be a link to said documentation here.

Reply:
The docs for individual services is in https://docs.weblate.org/en/latest/admin/machine.html